workspace/src/app/views/shared/TaskActivityOverview/TaskActivityOverview.tsx

```diff
@@ -105,6 +113,24 @@ export function TaskActivityOverview({ projectId, taskId }: IProps) {
         );
     };
 
+    const statusMapDependency = Array.from(activityStatusMap.values())
+        .map((v) => `${v.lastUpdateTime}-${v.concreteStatus}-${v.progress}`)
+        .join("|");
+
+    React.useEffect(() => {
+        const currentlyRunningActivity = Array.from(activityStatusMap.values()).find((a) => a.isRunning);
```

Comment:
If there is already ANY running activity, e.g. an hour long execution of a transform task, the cache update won't start at all. This can probably be solved the point below to start all cache activities at once.